appcues/src/test/java/com/appcues/logging/LogcuesTest.kt

```diff
@@ -1,81 +1,62 @@
 package com.appcues.logging
 
-import android.util.Log
-import com.appcues.LoggingLevel.INFO
-import com.appcues.LoggingLevel.NONE
-import io.mockk.called
-import io.mockk.every
-import io.mockk.mockkStatic
-import io.mockk.verify
-import org.junit.Before
+import app.cash.turbine.test
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.runBlocking
 import org.junit.Test
 
 internal class LogcuesTest {
 
-    @Before
-    fun setUp() {
-        mockkStatic(Log::class)
-        every { Log.i("Appcues", "message") } returns 1
-        every { Log.e("Appcues", "message") } returns 1
-    }
+    private val logcues = Logcues(Dispatchers.Unconfined)
 
     @Test
-    fun `info SHOULD log message WHEN level is INFO`() {
-        // GIVEN
-        val logcues = Logcues(loggingLevel = INFO)
-        // WHEN
-        logcues.info("message")
-        // THEN
-        verify { Log.i("Appcues", "message") }
-    }
-
-    @Test
-    fun `info SHOULD not log WHEN level is NONE`() {
-        // GIVEN
-        val logcues = Logcues(loggingLevel = NONE)
+    fun `info SHOULD emit info message`() = runBlocking {
         // WHEN
         logcues.info("message")
         // THEN
-        verify { Log::class.java wasNot called }
+        logcues.messageFlow.test {
```

Comment:
ah this is the [turbine](https://github.com/cashapp/turbine) update, cool

Reply:
yeah I guess its the first time we are actually testing a flow, turbine makes it easy